CHANGELOG.rst

```diff
@@ -2,6 +2,39 @@
 Change history
 ==============
 
+0.5.0 (2025-07-02)
+------------------
+
+.. warning::
+
+    The logging format has been changed from unstructured to structured with `structlog <https://www.structlog.org/en/stable/>`_.
+    For more information on the available log events and their context, see :ref:`manual_logging`.
+
+**New features**
+
+    * [:referentielijsten:`88`] Add structlog for observability
+    * [:referentielijsten:`89`] Enabled dark/light theme toggle in the Django Admin interface.
+    * [:open-api-framework:`139`] Integrate django-upgrade-check for easier Django upgrades and compatibility.
```

Comment:
Let's move this to `**Project maintenance**`, since it's not really a feature for the people using Referentielijsten, but more a useful thing for us as maintainers